components/playlist/browser/resources/playlist.tsx

```diff
@@ -22,18 +27,19 @@ import store from './store'
 
 function initialize () {
   render(
-    <Provider store={store}>
-      <BraveCoreThemeProvider
-        dark={DarkTheme}
-        light={Theme}
-      >
-        <App />
-      </BraveCoreThemeProvider>
-    </Provider>,
-    document.getElementById('root'))
+    <BrowserRouter>
+      <Provider store={store}>
+        <BraveCoreThemeProvider dark={DarkTheme} light={Theme}>
```

Comment:
Does this version of playlist (v1) actually support theming? If not you *might* be able to remove this :laughing: Leo theming doesn't use this at all.

Reply:
Yeah, that's true. I don't think I've used it at all 😄